clients/src/main/java/org/apache/kafka/clients/admin/CreateTopicsResult.java

```diff
@@ -94,7 +94,7 @@
         return futures.get(topic).thenApply(TopicMetadataAndConfig::replicationFactor);
     }
 
-    static class TopicMetadataAndConfig {
+    public static class TopicMetadataAndConfig {
```

Comment:
This still needs to be public: I looked at other similar objects in XXXResult in `admin`.